### PR TITLE
[SPARK-31871][CORE][WEBUI][2.4] Display the canvas element icon for sorting column

### PR DESCRIPTION
**Pull request by liucht-inspur**
What changes were proposed in this pull request?
issue link
https://issues.apache.org/jira/browse/SPARK-31871
this PR link 
https://github.com/apache/spark/pull/28680
As @sarutak suggested I open a new PR  for branch-2.4
This issue is resolved the sorted list cannot display ICONS
Before：
Executors Page
image
History Server Page
image

After：
Executors Page
image
History Server Page
image

Why are the changes needed?
The icon could not find the correct path

Does this PR introduce any user-facing change?
No.

How was this patch tested?
Fixed existing test.
history server page:

### CODE REVIEW

**Comment by sarutak** [comment]
ok to test.

**Comment by sarutak** [comment]
Hi @liucht-inspur , thank you for the contribution.
If you open a PR for a branch besides `master` , please add a proper tag to the title like `[2.4]` or `[3.0]`.
Please also see the [contribution guideline](https://spark.apache.org/contributing.html).

**Comment by sarutak** [comment]
retest this please.

**Comment by liucht-inspur** [comment]
> Hi @liucht-inspur , thank you for the contribution.
> If you open a PR for a branch besides `master` , please add a proper tag to the title like `[2.4]` or `[3.0]`.
> Please also see the [contribution guideline](https://spark.apache.org/contributing.html).

Ok, Thanks for your reminding, I will pay more attention next time

File: core/src/main/resources/org/apache/spark/ui/static/webui-dataTables.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+table.dataTable thead .sorting { background: url('images/sort_both.png') no-repeat bottom right; }
+
+table.dataTable thead .sorting_asc { background: url('images/sort_asc.png') no-repeat bottom right; }
+
+table.dataTable thead .sorting_desc { background: url('images/sort_desc.png') no-repeat bottom right; }
```
Comment (sarutak) [inline core/src/main/resources/org/apache/spark/ui/static/webui-dataTables.css:22]: nit: Could you append a blank line?

**Review by sarutak — COMMENTED** [review]
LGTM except one minor comment.

**Comment by liucht-inspur** [comment]
cc @sarutak and @HyukjinKwon

**Comment by sarutak** [comment]
@liucht-inspur Could you reflect my last comment?

File: core/src/main/resources/org/apache/spark/ui/static/webui-dataTables.css (hunk 1 above)
Comment (liucht-inspur) [reply core/src/main/resources/org/apache/spark/ui/static/webui-dataTables.css:22]: Add just one blank line at the end?

Comment (sarutak) [reply core/src/main/resources/org/apache/spark/ui/static/webui-dataTables.css:22]: Yes, as the warning suggests.
end-of-line

**Comment by liucht-inspur** [comment]
retest this please.

**Comment by sarutak** [comment]
retest this please.

**Comment by sarutak** [comment]
@liucht-inspur You can't run Jenkins by yourself for now.
If you continue to contribute to Spark, you will added to the whitelist and be given the right to run Jenkins by saying "retest this please".

**Comment by sarutak** [comment]
retest this please.

**Review by sarutak — APPROVED** [review]

**Comment by sarutak** [comment]
Merged to `branch-2.4`.
Thanks @liucht-inspur !

**Pull request closed without merging**